doc/releases/release-notes-3.6.rst

```diff
@@ -246,6 +246,22 @@ Drivers and Sensors
 * Ethernet
 
   * The "native_posix" ethernet driver now supports being built with embedded C libraries.
+  * Enabled HW checksum offloading for STM32H7.
+  * Added implementation of Open Alliance's TC6 T1S driver.
+  * Added xmc4xxx driver.
+  * Added NXP enet driver with PTP support.
+  * Added KSZ8081 PHY driver.
+  * Added proper IPv4 multicast support to NXP mcux driver.
+  * Added LAN8651 T1S support.
+  * Added DSA support to STM32.
+  * Added tja1103 PHY support.
+  * Added Nuvoton numaker support.
+  * Fixed lan865x driver.
+  * Fixed s32_gmac driver.
+  * Fixed phy_mii driver.
+  * Fixed sam_gmac driver.
+  * Fixed adin2111 driver.
+  * Fixed ksz8081 driver.
```

Comment:
It would be nice to mention what was fixed in these drivers

Reply:
+1, I was going to say exactly the same thing :) Bonus points if you could also use the `:dtcompatible:` role to create nice links to the documentation of the bindings.

Reply:
I might cherry pick that suggestion too

Reply:
In 3.5 I added this kind of data and I got notes that it was too much data I had here. So for 3.6 tried to avoid this. There are multiple changes for each driver so I am just wondering what to put here.

Reply:
In general you don't need to mention bug fixes here, as they will be summarized based on which bug issues were fixed for this release at the bottom. 

If you do mention them, I think we need a bit more information than what is currently provided in order for it to add any value for the reader.

Reply:
> In general you don't need to mention bug fixes here, as they will be summarized based on which bug issues were fixed for this release at the bottom. 

I think we dropped this for 3.5, right?
I wouldn't necessarily be against re-introducing this *but* it would need some work (and arguably we don't have enough metadata capture to generate an accurate representation of what was *really* fixed)

- issues closed as "not planned" should not be part of the list. IIRC they were included and that's really the opposite of what we want.
- as part of triaging, we may close issues as "fixed a long time ago", but we're not capturing this information, so the list is also full of things that don't really have anything to do with the release. I wonder if it would make sense to make the milestone field mandatory (if that's even possible...)?